src/core/gltf2importer/gltf2exporter.h

```diff
@@ -61,7 +61,10 @@ public Q_SLOTS:
     void setContext(GLTF2Context *context);
     void setScene(SceneEntity *scene);
 
-    QJsonObject compress(QDir folder, QJsonObject rootObject);
+    QJsonObject compress(
+            QDir sourceFolder,
+            QDir targetFolder,
+            QJsonObject rootObject);
```

Comment:
const QJsonObject &

Reply:
I also pass the QDir as const& now, is it ok ?